spec/models/campaign_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe Campaign, type: :model do
+  let(:campaign) { build(:campaign) }
+
+  context 'when title fields are empty' do
```

Comment:
I would wrap all the test cases with a single `describe 'validations' `block and remove context blocks, these `it` tests are quite self-descriptive and `contexts` don't add much sense
@okohuch what do you think?

Reply:
@vikabelous agree